### ISSUE TITLE
[bug] draging list creates item unexpectly on live demos

### ISSUE THREAD

**Issue by SimonLeeee** [SortableJS/vue.draggable.next#114]
Please check the https://sortablejs.github.io/vue.draggable.next/#/clone
First step, move the left list item John to the bottom of right list.
Second step, do the first step again.
Finally, in right list, move the first item to the bottom, then you will see right list create an item unexpected.

**Comment by zdimaz** [comment]
Disables yours extantions )!

**Comment by Deepx-China** [comment]
I also have this problem

**Comment by AIDeepx** [comment]
You can refer to cust clone demo, use `clone` method

**Comment by SimonLeeee** [comment]
> You can refer to cust clone demo, use `clone` method

Yeah it can be an alternate way, but problem still remains, the live demos need a fix

**Comment by Zedicius** [comment]
Yeah, I'm having this issue on my website.  It's not just with cloning lists either but simple lists as well.  The issue does appear to be visual because the actual data list remains the same so the component is generating too many dom elements for some reason.  Looking in too it.

**Comment by Zedicius** [comment]
Ah, found it.  It's because the item-key attribute in the draggable component refers too 'name'.  If you put two 'Johns' in the list, the item-key is no longer unique and the list gets messed up.  As long as you make sure that the item-key attribute is a unique value things seem to work fine.
